src/DynamoCore/Core/IDSDKManager.cs

```diff
@@ -78,6 +78,18 @@ public string Username
             }
         }
 
+        /// <summary>
+        /// Gets the userid of the logged in user.
+        /// </summary>
+        internal string UserId
+        {
+            get
+            {
+                var result = IDSDK_GetUserInfo();
+                return result != null ? result.UserId : String.Empty;
```

Comment:
if the user id is an empty string or null - should we still be sending data? How will the backend interpret that?